app/edge/devices/components/AMTDevicesDatatable/columns/actions.tsx

```diff
@@ -0,0 +1,83 @@
+import {CellProps, Column, TableInstance} from 'react-table';
+import {useRouter} from "@uirouter/react";
+import { Device } from "Portainer/hostmanagement/open-amt/model";
+import { MenuItem } from "@reach/menu-button";
+import { ActionsMenu } from "Portainer/components/datatables/components/ActionsMenu";
```

Comment:
should we be mixing camelCase and PascalCase for the imports?

Reply:
For some reason my commit didn't execute the eslint scripts, I, fixed most of this imports comments